doc/conftest.py

```diff
@@ -91,6 +99,8 @@ def pytest_runtest_setup(item):
         setup_compose()
     elif IS_PYPY and fname.endswith('modules/feature_extraction.rst'):
         raise SkipTest('FeatureHasher is not compatible with PyPy')
+    elif fname.endswith('datasets/loading_other_datasets.rst'):
+        setup_impute()
```

Comment:
Should this be ``setup_loading_other_dataset``?

How did you run into the error? Is this for building the documentation without pandas installed?

Reply:
> Should this be `setup_loading_other_dataset`?

Argh , yes! Sorry!

> How did you run into the error? Is this for building the documentation without pandas installed?

I was investigating a segfault in an environment different from my local environment: I have created a conda environment from scratch and ran the tests. I forgot to install pandas and I had a failing test with loading_other_datasets.